frontend/package.json

```diff
@@ -224,15 +236,18 @@
     "@graphql-codegen/typescript-graphql-files-modules": "^1.15.1",
     "@graphql-codegen/typescript-operations": "^1.15.1",
     "@pmmmwh/react-refresh-webpack-plugin": "^0.5.15",
-    "@testing-library/jest-dom": "^6.6.3",
-    "@testing-library/react": "^12.0.0",
-    "@testing-library/user-event": "^13.1.9",
+    "@swc/core": "^1.15.3",
+    "@swc/jest": "^0.2.39",
+    "@testing-library/dom": "^10.4.1",
+    "@testing-library/jest-dom": "^6.9.1",
+    "@testing-library/react": "^12.1.4",
```

Comment:
We could consider bumping the react-testing-library and its dependencies up to the latest version later on.

Reply:
See https://issues.redhat.com/browse/CONSOLE-4510